app/src/main/res/values-night/color_palette.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="primary_color">@color/oppia_primary_2</color>
+  <color name="primary_dark_color">@color/oppia_primary_dark_2</color>
+  <color name="accent_color">@color/oppia_accent_2</color>
+  <color name="primary_text_color">@color/oppia_white_subtext</color>
+  <color name="heading_text_color">@color/white</color>
+  <color name="add_profile_background_color">@color/oppia_background_black</color>
+  <color name="switch_description_color">@color/oppia_white_subtext</color>
+  <color name="edit_text_input_background">@color/oppia_edit_text_background</color>
+  <color name="add_profile_switch_text_color">@color/white</color>
+  <color name="text_input_layout_error_color">@color/error_red</color>
```

Comment:
These are component colors (they're tied to specific components) and thus shouldn't be defined here.

Reply:
Done.